es6-promises/index.js

```diff
@@ -0,0 +1,9 @@
+const takeAChance = require('./take-a-chance');
+
+const promise = takeAChance('Henry Nichols');
+
+promise.then(message => {
+  console.log(message)
+}).catch(message => {
+  console.log(message)
+});
```

Comment:
Try only logging the message property of the error object. Your results should look like the example.